features/bootstrap/FeatureContext.php

```diff
@@ -21,7 +22,6 @@ class FeatureContext implements Context {
    * @beforeNodeCreate
    */
   public static function alterNodeParameters(BeforeNodeCreateScope $scope) {
-    call_user_func('parent::alterNodeParameters', $scope);
```

Comment:
This line should remain or it won't resolve node creation timestamps correctly under Drupal 8. Seems we have no tests that deal with timestamps on D8 :-/